Kickstarter-iOS/MessageBannerType.swift

```diff
@@ -9,24 +8,20 @@ public enum MessageBannerType {
   var backgroundColor: UIColor {
     switch self {
     case .success:
-      return UIColor.ksr_cobalt_500
+      return .ksr_cobalt_500
     case .error:
-      return UIColor.ksr_apricot_600
+      return .ksr_apricot_600
     case .info:
-      return UIColor.ksr_cobalt_500
+      return .ksr_cobalt_500
     }
   }
 
-  var iconImage: UIImage? {
+  var iconImageName: String? {
```

Comment:
Returning the string name here so that this doesn't need access to the `image(named:)` function.